prospector/datamodel/advisory.py

```diff
@@ -123,15 +123,28 @@ def extract_path_tokens(text: str) -> "list[str]":
     Returns:
         list: a list of paths that are found
     """
-    return [
-        re.split(r"\.|,|/", token.rstrip(r".,;:?!\"'"))
-        for token in text.split(" ")
-        if ("/" in token.rstrip(r".,;:?!\"'") and not token.startswith("</"))
-        or (
-            "." in token.rstrip(r".,;:?!\"'")
-            and token.rstrip(r".,;:?!\"'").split(".")[-1] in RELEVANT_EXTENSIONS
+    tokens = re.split(r"\s+", text)  # split the text into words
+    tokens = [
+        token.strip(",.:;-+!?)]}'\"") for token in tokens
+    ]  # removing common punctuation marks
+    paths = []
+    for token in tokens:
+        is_contains_path_separators = ("\\" in token) or ("/" in token)
```

Comment:
better name: `contains_path_...` or `does_contain_path_...`

Reply:
I prefer (not always succeded) to call my guarding or boolean variables like is_* since I had too much naming conflict with other concepts in the past, like regex matches, pattern storing classes, preprocessing method, etc. But of course, I could accommodate your suggested names if that is your wish. I just wish to clarify my logic which could be faulty 😉.

Reply:
Valid point, but we have `has_relevant_extension` anyway :-)

I think that nowadays, with proper IDE support, the `is_....` convention can be relaxed in favour of more readable (from a grammar perspective) names.